docs/modules/commons/pages/table-transformers.adoc

```diff
@@ -516,3 +516,77 @@ Examples:
 |value |1    |
 |value2|0    |
 ----
+
+== INNER_JOIN
+
+`INNER_JOIN` transformer combines rows from two tables whenever there are matching values between the columns.
+
+[cols="1,3", options="header"]
+|===
+|Parameter
+|Description
+
+|`leftTableJoinColumn`
+|the name of the column in the first table for matching
+
+|`rightTableJoinColumn`
+|the name of the column in the second table for matching
+
+|`tables`
+|paths to examples tables or transformers to inner join
+|===
+
+[IMPORTANT]
+====
+* number of unique tables paths must be equal 2
```

Comment:
it's not required if one table is specified as body under transformer

Reply:
also it's not clear which table is 'left' and which is 'right'

Reply:
changed